src/zorak/cogs/admin/admin_automod_spam_messages.py

```diff
@@ -0,0 +1,150 @@
+"""
+A listener that looks for repeat messages and destroys them.
+"""
+import logging
+from datetime import datetime, timedelta
+
+import discord.errors
+from discord.ext import commands
+
+from zorak.utilities.cog_helpers._embeds import (
+    embed_spammer,  # pylint: disable=E0401
+    embed_spammer_warn  # pylint: disable=E0401
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+def is_second_message(first, second):
```

Comment:
pedantic; unneeded abstraction

Reply:
True, there were 2 iterations before this and this might be some leftover.

Reply:
Lol, wasn't even used in the code. Was just old floating trash